tests/openfhe/end_to_end/simple_sum_test.cpp

```diff
@@ -19,17 +19,23 @@ namespace heir {
 namespace openfhe {
 
 TEST(BinopsTest, TestInput1) {
-  CCParams<CryptoContextBGVRNS> parameters;
-  parameters.SetMultiplicativeDepth(2);
-  parameters.SetPlaintextModulus(65537);
-  CryptoContext<DCRTPoly> cryptoContext = GenCryptoContext(parameters);
-  cryptoContext->Enable(PKE);
-  cryptoContext->Enable(KEYSWITCH);
-  cryptoContext->Enable(LEVELEDSHE);
+  // CCParams<CryptoContextBGVRNS> parameters;
```

Comment:
nit: could you please remove the commented out stubs in the test harnesses? ty!

Reply:
sure!